tiled/server/core.py

```diff
@@ -375,10 +380,10 @@ async def construct_data_response(
     try:
         if filter_for_access is not None:
             content = await ensure_awaitable(
-                serializer, payload, metadata, filter_for_access
+                serializer, format, payload, metadata, filter_for_access
```

Comment:
The fact that this passes tests reveals a gap in our test suite. Reading the code above, this needs to be `media_type`, the chosen match between the (list of) types provided by the user via a `format` query parameter or an `Accept` header and the set of types supported by the server.